### PR TITLE
SRVKE-1523: [release-v1.10] Re-add removed HPA for ingress and filter

### PR DESCRIPTION
**Pull request by pierDipi**
As per title

It re-adds the upstream HPAs for ingress and filter deployments

### CODE REVIEW

File: openshift/release/artifacts/eventing-core.yaml [hunk 1]
```diff
@@ -1247,15 +1247,33 @@ metadata:
...
     registry.knative.dev/eventTypes: |
       [
-        { "type": "dev.knative.apiserver.resource.add" },
-        { "type": "dev.knative.apiserver.resource.delete" },
-        { "type": "dev.knative.apiserver.resource.update" },
-        { "type": "dev.knative.apiserver.ref.add" },
-        { "type": "dev.knative.apiserver.ref.delete" },
-        { "type": "dev.knative.apiserver.ref.update" }
+        {
+          "type": "dev.knative.apiserver.resource.add",
```
Comment (skonto) [inline openshift/release/artifacts/eventing-core.yaml:1254]: I guess these are the latest changes from upstream?

Comment (pierDipi) [reply openshift/release/artifacts/eventing-core.yaml:1254]: No, we missed this codegen from a previous PR and codegen is not verified

**Review by creydr — APPROVED** [review]
/lgtm

**Comment by skonto** [comment]
@pierDipi should we unhold?

**Pull request merged**